savant/deepstream/pipeline.py

```diff
@@ -147,13 +147,16 @@ def __init__(
 
         # nvjpegdec decoder is selected in decodebin according to the rank, but
         # there are problems with the plugin:
-        # 1) https://forums.developer.nvidia.com/t/nvvideoconvert-memory-compatibility-error/226138;
-        # 2) jpeg to png conversion gives incorrect alpha channel;
-        # 3) memory type mismatch, even though we use the same
-        #  nvbuf-memory-type for nvvideoconvert and nvstreammux downstream
+        # 1) nvjpegdec doesn't support some jpg
+        #  https://forums.developer.nvidia.com/t/nvvideoconvert-memory-compatibility-error/226138;
+        # 2) nvjpegdec produces incorrect alpha channel
+        #  https://forums.developer.nvidia.com/t/nvjpegdec-produces-transparent-frames/223005
```

Comment:
Is this issue still present on DS 6.3? They said it was fixed https://forums.developer.nvidia.com/t/nvjpegdec-produces-transparent-frames/223005/16